.github/workflows/list-test.yml

```diff
@@ -0,0 +1,129 @@
+name: Linting and testing
```

Comment:
Nitpick, but there is a typo in the filename (`list-test.yml`)

Reply:
Fixed, thanks!